### PR TITLE
[module.global.frag] Simplify wording

### PR DESCRIPTION
**Pull request by lprv**
* Simplify the wording of [[module.global.frag]/3.5.2](https://eel.is/c++draft/module.global.frag#3.5.2)
* Make the phrasing of 'one of *D* and *M*' consistent across following bullets

### CODE REVIEW

**Comment by tkoeppe** [comment]
@opensdh Would you have thoughts on whether this is a useful (and correct) simplification?

**Comment by opensdh** [comment]
Seems right to me; I think the modules wording predates having a good definition of "declare the same entity", as silly as that sounds.

**Review by jensmaurer — APPROVED** [review]
Looks good.

**Comment by tkoeppe** [comment]
Thanks, everyone!
